src/AppBundle/Command/AddUserCommand.php

```diff
@@ -298,4 +292,66 @@ private function getCommandHelp()
 
 HELP;
     }
+
+    /**
+     * Gets the value of entityManager.
+     *
+     * @return ObjectManager
+     */
+    public function getEntityManager()
+    {
+        return $this->entityManager;
+    }
+
+    /**
+     * Sets the value of entityManager.
+     *
+     * @param ObjectManager $entityManager the entity manager
+     *
+     * @return self
+     */
+    public function setEntityManager(ObjectManager $entityManager)
```

Comment:
IMO, you should use constructor injection here (or the new experimental getter overriding once migrating to 3.3 if we want to keep the dependency lazy, as this is a good use case)